bladex/params.py

```diff
@@ -0,0 +1,348 @@
+            then an array of zeros is assigned, in which its length is equal
+            to the radii array.
+        - Any array that is not numpy is converted to be one.
+        - In case the user specifies the parameter array values but not with
+            same length, then an exception is raised.
+        """
+        if self.radii is None:
+            raise ValueError('Array radii can not have a None value.')
+
+        if not isinstance(self.radii, np.ndarray):
+            self.radii = np.asarray(self.radii)
+
+        # If any parameter but radii is not inserted then assign that parameter
+        # with array of zeros. This is useful in case the user is interested in
+        # only one or few parameters, while not interested in (or does have
+        # information about) the remaining ones.
+        if self.chord is None:
+            self.chord = np.zeros(self.radii.shape[0])
+        if self.pitch is None:
+            self.pitch = np.zeros(self.radii.shape[0])
+        if self.rake is None:
+            self.rake = np.zeros(self.radii.shape[0])
+        if self.skew is None:
+            self.skew = np.zeros(self.radii.shape[0])
+        if self.camber is None:
+            self.camber = np.zeros(self.radii.shape[0])
+
+        # In case inserted parameters are not numpy arrays
+        if not isinstance(self.chord, np.ndarray):
+            self.chord = np.asarray(self.chord)
+        if not isinstance(self.pitch, np.ndarray):
+            self.pitch = np.asarray(self.pitch)
+        if not isinstance(self.rake, np.ndarray):
+            self.rake = np.asarray(self.rake)
+        if not isinstance(self.skew, np.ndarray):
+            self.skew = np.asarray(self.skew)
+        if not isinstance(self.camber, np.ndarray):
+            self.camber = np.asarray(self.camber)
+
+        # check the case if user inserts inhomogeneous radial distributions
+        if not self.chord.shape == self.radii.shape:
+            raise ValueError('Array chord must have same shape of array radii.')
+        if not self.pitch.shape == self.radii.shape:
+            raise ValueError('Array pitch must have same shape of array radii.')
+        if not self.rake.shape == self.radii.shape:
+            raise ValueError('Array rake must have same shape of array radii.')
+        if not self.skew.shape == self.radii.shape:
+            raise ValueError('Array skew must have same shape of array radii.')
+        if not self.camber.shape == self.radii.shape:
+            raise ValueError(
+                'Array camber must have same shape of array radii.')
+
+        # If inserted deformations array not correspond to nbasis
+        for param in ['chord', 'pitch', 'rake', 'skew', 'camber']:
+            if not self.nbasis[param] == len(self.deformations[param]):
+                raise ValueError(
+                    param + ' deformations must correspond to nbasis.')
+
+    def _param_pptc(self):
```

Comment:
This should not be here. If you want you can create this particular parameter file and put it in a folder so it can be read.